examples/se_2_3_localization.cpp

```diff
@@ -0,0 +1,412 @@
+
+    // acceleration due to gravity in world frame
+    Vector3d g;
+    g << 0, 0, -9.80665;
+    const double dt = 0.1;
+
+    // IMU measurements in IMU frame
+    Vector3d alpha, omega, alpha_prev, omega_prev;
+    alpha << 10.0, 0.0, 9.80665; // constant acceleration along x-direction in IMU frame
+    omega << 1, 0.1, 0; // constant angular velocity about x- and y-direction in IMU frame
+    // Previous IMU measurements in IMU frame initialized to values expected when stationary
+    alpha_prev << 0.0, 0.0, 9.80665;
+    omega_prev << 0, 0, 0;
+
+    // Define a control vector and its noise and covariance
+    manif::SE_2_3Tangentd  u_simu, u_est, u_unfilt;
+    Vector9d            u_nom, u_noisy, u_noise;
+    Array9d             u_sigmas;
+    Matrix9d            U;
+
+    u_sigmas << 0.0, 0.0, 0.0, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1;
+    U        = (u_sigmas * u_sigmas).matrix().asDiagonal();
+
+    // Declare the Jacobians of the motion wrt robot and control
+    manif::SE_2_3d::Jacobian J_x, J_u;
+
+    // Define five landmarks in R^3
+    Vector3d b0, b1, b2, b3, b4, b;
+    b0 << 2.0,  0.0,  0.0;
+    b1 << 3.0, -1.0, -1.0;
+    b2 << 2.0, -1.0,  1.0;
+    b3 << 2.0,  1.0,  1.0;
+    b4 << 2.0,  1.0, -1.0;
+    std::vector<Vector3d> landmarks;
+    landmarks.push_back(b0);
+    landmarks.push_back(b1);
+    landmarks.push_back(b2);
+    landmarks.push_back(b3);
+    landmarks.push_back(b4);
+
+    // Define the beacon's measurements
+    Vector3d                y, y_noise;
+    Array3d                 y_sigmas;
+    Matrix3d                R;
+    std::vector<Vector3d>   measurements(landmarks.size());
+
+    y_sigmas << 0.01, 0.01, 0.01;
+    R        = (y_sigmas * y_sigmas).matrix().asDiagonal();
+
+    // Declare the Jacobian of the measurements wrt the robot pose
+    Matrix<double, 3, 9>    H;      // H = J_e_x
+
+    // Declare some temporaries
+    Vector3d                e, z;   // expectation, innovation
+    Matrix3d                E, Z;   // covariances of the above
+    Matrix<double, 9, 3>    K;      // Kalman gain
+    manif::SE_2_3Tangentd      dx;     // optimal update step, or error-state
+    manif::SE_2_3d::Jacobian   J_xi_x; // Jacobian is typedef Matrix
+    Matrix<double, 3, 9>       J_e_xi; // Jacobian
+
+    //
+    //
+    // CONFIGURATION DONE
+
+
+
+    // DEBUG
+    cout << std::fixed   << std::setprecision(3) << std::showpos << endl;
+    cout << "X STATE     :    X      Y      Z    TH_x   TH_y   TH_z     V_x   V_y    V_z" << endl;
+    cout << "---------------------------------------------------------------------------" << endl;
+    cout << "X initial   : " << X_simulation.log().coeffs().transpose() << endl;
+    cout << "---------------------------------------------------------------------------" << endl;
+    // END DEBUG
+
+
+
+
+    // START TEMPORAL LOOP
+    //
+    //
+
+    // Make 10 steps. Measure up to three landmarks each time.
+    for (int t = 0; t < 10; t++)
```

Comment:
I think `t` and `dt` should be consistent. Here, `t` is now a counter, thus without units, and `dt` is in seconds. I suggest replacing this for() by
```
for (double t = 0; t < 10; t += dt)
```

Reply:
the `10` can be changed by whatever suits you best